packages/magento-product-bundle/components/BundleCartItem/BundleCartItem.graphql

```diff
@@ -1,4 +1,9 @@
 fragment BundleCartItem on BundleCartItem @inject(into: ["CartItem"]) {
+  prices {
+    price {
+      currency
+    }
+  }
```

Comment:
I don't think this something we'd want? Can you explain what you want to do with the currency field?